app/models/game.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'titlecase'
+
+class Game < ApplicationRecord
+  belongs_to :user
+  has_many :shopping_lists, dependent: :destroy
+
+  validates :name, uniqueness: { scope: :user_id },
+                   format: {
+                     with: /\A\s*[a-z0-9 '\,]*\s*\z/i,
+                     message: "can only contain alphanumeric characters, spaces, commas (,), and apostrophes (')"
+                   }
+
+  before_save :format_name
+
+  def aggregate_shopping_list
+    shopping_lists.find_by(aggregate: true)
+  end
+
+  private
+
+  def format_name
```

Comment:
There is a bug in this method but it already exists in the `ShoppingList` model in the `format_title` method, so I've made a card to fix the issues at the same time before this feature is shipped.